lib/spree_multi_currency/engine.rb

```diff
@@ -6,7 +6,7 @@ class Engine < Rails::Engine
 
     require 'spree/core/currency_helpers'
 
-    def self.activate
+    def self.activate(app)
```

Comment:
Unused method argument - `app`. If it's necessary, use `_` or `_app` as an argument name to indicate that it won't be used. You can also write as `activate(*)` if you want the method to accept any arguments but don't care about them.